src/Apps/Collect/Routes/Collection/Components/Header/__tests__/fixtures/artworks.ts

```diff
@@ -1,4 +1,4 @@
-import { DefaultHeader_headerArtworks } from "__generated__/DefaultHeader_headerArtworks.graphql"
+// import { DefaultHeader_headerArtworks } from "__generated__/DefaultHeader_headerArtworks.graphql"
```

Comment:
Woops, yeah I could probably use a quick pair session on updating fragments and types for fixtures.